config/config_helpers.go

```diff
@@ -204,20 +206,14 @@ func getParentTfVarsDir(include *IncludeConfig, terragruntOptions *options.Terra
 
 func parseGetEnvParameters(parameters string) (EnvVar, error) {
 	envVariable := EnvVar{}
-	matches := HELPER_FUNCTION_GET_ENV_PARAMETERS_SYNTAX_REGEX.FindStringSubmatch(parameters)
-	if len(matches) < 2 {
-		return envVariable, errors.WithStackTrace(InvalidGetEnvParams(parameters))
-	}
+	parsed := parseParamList(parameters)
 
-	for index, name := range HELPER_FUNCTION_GET_ENV_PARAMETERS_SYNTAX_REGEX.SubexpNames() {
-		if name == "env" {
-			envVariable.Name = strings.TrimSpace(matches[index])
-		}
-		if name == "default" {
-			envVariable.DefaultValue = strings.TrimSpace(matches[index])
-		}
+	if len(parsed) != 2 || len(parsed[0]) == 0 {
```

Comment:
Then its just a matter of checking that we have the right number of parameters.

Incidentally, the original `HELPER_FUNCTION_GET_ENV_PARAMETERS_SYNTAX_REGEX` also validated that the first parameter is non-empty.  So we do that here.